app/views/layouts/application.html.erb

```diff
@@ -1,36 +1,31 @@
 <!DOCTYPE html>
-<html lang="en" class="container">
+<html lang="en">
   <head>
-    <title>OSS Friday</title>
+    <title>Open Source Friday</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <% if Rails.env.development? %>
+    <% if false && Rails.env.development? %>
```

Comment:
Annoying and made page reloads slow because I was using bad wifi at the time. I'm fine re-enabling it before this branch wraps up.